### ISSUE TITLE
A11y: [Be in your element] “Welcome to the fastest Element Pro ever” has not enough contrast with the BG

### ISSUE THREAD

**Issue by bmarty** [element-hq/element-x-android#7156]
- **Checkpoint ID:** 11.1.4.3/1.1
- **Component:** Be in your element

### Issue Description

The page background uses a gradient. As a result, individual letters in the sentences
“Welcome to the fastest Element Pro ever” and “Supercharged for speed and simplicity” fall
below the text contrast requirements.

### Recommendation

--- 
*Report Date: 25.02.2026*

**Comment by bmarty** [comment]
I am quickly checking the code and it should be nice to have a updated Figma first.

The reference is https://www.figma.com/design/0MMNu7cTOzLOlWb7ctTkv3/Element-X?node-id=1816-97419, but I see that some iOS font are used, and colors are not semantic colors, but sometimes hard-coded colors.

**Comment by americanrefugee** [comment]
I've updated the text content to include "Pro", and changed the text color to tex/primary.

- [iOS](https://www.figma.com/design/jvZ7TIyd7V9Vn8sIDB9vMd/Compound-iOS-Templates?node-id=4032-209714&t=rNOOPNMlJ6BxDRcW-4)
- [Android](https://www.figma.com/design/pDlJZGBsri47FNTXMnEdXB/Compound-Android-Templates?node-id=41-6503&t=nWHwh92IHC3z4Dif-4)

<img width="393" height="852" alt="Image" src="https://github.com/user-attachments/assets/294b0a4e-3b40-40d0-9e61-6067ebb2f96d" />

<img width="360" height="800" alt="Image" src="https://github.com/user-attachments/assets/19d136a3-2625-44e9-9a26-923efa945d22" />
